Cargo.toml

```diff
@@ -17,3 +17,5 @@ serde = "1.0"
 ring = { version = "0.16.5", features = ["std"] }
 base64 = "0.10"
 chrono = "0.4"
+pem = "0.6"
```

Comment:
hmm this add `failure` to the list of sub-dependencies, I'll check if the author would accept a PR removing it.

Reply:
> How did you generate the PEM keys? I will add all the defaut ones from jwt.io later on to ensure everything is fine.

All of the key data is derived from what was already in the repo. I renamed the `private_rsa_key.pem` to its corresponding `_pkcs8` type. I've gone through my zsh history for exact answers.

The PKCS#1 keys are the base64 encoded `.der` keys already in the repository with the corresponding PEM tag `PRIVATE RSA KEY` or `PUBLIC RSA KEY`.

The PKCS#8 RSA public key was created from the private key with an openssl command.
`openssl pkcs8 -topk8 -in private_rsa_key.pem -out private_rsa_key_pkcs8.pem -nocrypt`

The PKCS#8 EC private key was created from the `.pk8` file already in the repository.
`base64 public_ecdsa_key.pk8 |  awk '{gsub(/.{64}/,"&\n")}1' > public_ecdsa_key.pem` followed by manually adding the PEM Tagging `-----BEGIN PRIVATE KEY-----` and `-----END PRIVATE KEY-----` around the existing file.

And the public key was then derived from that
`openssl ec -in private_ecdsa_key.pem -pubout -out public_ecdsa_key.pem`

Reply:
It's now 0.7 on crates and this is now updated.